### ISSUE TITLE
themes/book: Search bar has insufficient color contrast

### ISSUE THREAD

**Issue by kevinlin1** [jupyter-book/myst-theme#642]
When running a Lighthouse accessibility check, the book theme does not have enough contrast in the search bar. The "Search" placeholder text and keyboard shortcut iconography could be darker while the background could be lighter (or just plain white for maximum contrast).
